python/py-tlsh/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        trendmicro tlsh 3.11.0
+name                py-tlsh
+platforms           darwin
+license             Apache-2 BSD
+maintainers         nomaintainer
+description         Trend Micro Locality Sensitive Hash
+long_description    TLSH is a fuzzy matching library. Given a byte stream with \
+                    a minimum length of 50 bytes TLSH generates a hash value \
+                    which can be used for similarity comparisons.
+
+checksums           rmd160  95162351a0d04d6922bea9812acbe84a310ff7de \
+                    sha256  e36a4e8ea0670e8561af72be520d82c7eb38fd86f751dd5a5fa90b9df91d7519 \
+                    size    2763206
+
+dist_subdir         tlsh
+worksrcdir          ${distname}/py_ext
+
+python.versions     37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                        path:bin/cmake:cmake
```

Comment:
Would it make sense to use the make 1.1 portgroup?

Reply:
Yes, but it would conflict with the python portgroup. Any idea how to avoid the conflicts?

Reply:
In what way do they conflict exactly ?

Reply:
@l2dy: Is it because they both set the build command? Which one is the "correct" one in this case? Maybe add that portgroup 2nd and it will use the correct build command?

Reply:
I think I might I run into problems myself in the past trying to combine these two PGs. If what is here works I suggest leaving it for now, but then look into the cmake+python PG  issue separately. Ok ?